tests/auto/gltfexporter/tst_gltfexporter.cpp

```diff
@@ -59,6 +61,216 @@ class tst_GLTFExporter : public QObject
 
 private Q_SLOTS:
 
+    void checkEmbedAll()
```

Comment:
maybe check which files are generated. In particular that in the case of embedded there's no .bin file

Reply:
done - there was already a check but it wasn't enough in the case where an external binary buffer was used, and only partly stripped after draco compression (if e.g. it contained both a mesh and a texture)